pmda/rmsf.py

```diff
@@ -186,13 +187,13 @@ def _reduce(res, frame_result):
         # res[0]  sum of squares
         # res[1]  mean
         """
-        k = frame_result[0]
-        position = frame_result[1].positions
+        k = result_single_frame[0]
+        position = np.float64(result_single_frame[1].positions)
```

Comment:
Nicer numpy idiom:
```python
positions = result_single_frame[1].positions.astype(np.float64)
```
Also name the variable `positions` like the attribute itself.